site/docs/v1/tech/oauth/_logout_behavior_all_applications.adoc

```diff
@@ -0,0 +1 @@
+An Application's OAuth configuration allows you to specify the logout behavior of an application as either "All applications" or "Redirect only." Specifying the default value of "All applications" will call the registered logout URLs for each application on the Tenant _except_ for the FusionAuth admin application. See the https://github.com/FusionAuth/fusionauth-issues/issues/1699[GitHub issue] for more information.
```

Comment:
1. `"Redirect only."` -> `"Redirect only".`
2. `on the Tenant` -> `in the Tenant`

Reply:
[MLA docs](https://style.mla.org/the-placement-of-a-comma-or-period-after-a-quotation/).
> When a comma or period is needed after a quotation, publishers in the United States typically put the punctuation mark before the closing quotation mark.

British publishers put the period outside the quotation marks.

Do you still want me to change that?